### PR TITLE
feat(Progress): add new prop orientation to display vertically

### LINKED ISSUE #18761: Vertical progress bar
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
need of vertical progress bar

### What does the proposed API look like?
have something like a "orientation" prop for the progress bar

### PR DESCRIPTION
**Pull request by Larissagilliane**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Issue #18761 

### 💡 Background and solution

Turned able the usage of Progress Bar component with vertical orientation like mentioned at isse #18761 

Screen Shot 2019-09-20 at 15 28 57

To use just set the property "orientation" with "vertical" value, like "orientation='vertical' " to the Progress component.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Progress add new prop `orientation` to display vertically      |
| 🇨🇳 Chinese |     Progress 新增属性 `orientation` 以垂直显示      |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

Closes #18761 

-----
[View rendered README-zh_CN.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/README-zh_CN.md)
[View rendered README.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/README.md)
[View rendered components/progress/demo/orientation.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/components/progress/demo/orientation.md)
[View rendered components/progress/index.en-US.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/components/progress/index.en-US.md)
[View rendered components/progress/index.zh-CN.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/components/progress/index.zh-CN.md)
[View rendered components/tooltip/index.en-US.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/components/tooltip/index.en-US.md)
[View rendered components/tooltip/index.zh-CN.md](https://github.com/Larissagilliane/ant-design/blob/verticalProgress/components/tooltip/index.zh-CN.md)

### CODE REVIEW

**Review by luciamo — DISMISSED** [review]

**Comment by orzyyyy** [comment]
Rebase your commits for keeping the PR clean, it's hard to review now.

File: components/progress/index.en-US.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Properties that shared by all types.
 | percent | to set the completion percentage | number | 0 |  |
 | showInfo | whether to display the progress value and the status icon | boolean | true |  |
 | status | to set the status of the Progress, options: `success` `exception` `normal` `active`(line only) | string | - |  |
+| orientation | to set the orientation from horizontal to vertical, options: `default` `vertical` | string | - |  |
```
Comment (orzyyyy) [inline components/progress/index.en-US.md]: It's better to set `horizontal` as a default value, for `default` may make people confused.

**Comment by afc163** [comment]
Mixed unrelated commits.

File: components/progress/index.en-US.md (hunk 1 above)
Comment (Larissagilliane) [reply components/progress/index.en-US.md]: done

Comment (orzyyyy) [reply components/progress/index.en-US.md]: 1. Maybe you forget to update this in docs. `default` => `horizontal`
2. The penultimate grid can be filled in with default values, like 
| options: `horizontal` `vertical` | string | `horizontal` |  3.24.0   |

File: components/progress/index.zh-CN.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ title: Progress
 | percent | 百分比 | number | 0 |  |
 | showInfo | 是否显示进度数值或状态图标 | boolean | true |  |
 | status | 状态，可选：`success` `exception` `normal` `active`(仅限 line) | string | - |  |
+| orientation | 将方向从水平设置为垂直, 可选: `default` `vertical` | string | - |  |
```
Comment (orzyyyy) [inline components/progress/index.zh-CN.md]: 1. Maybe you forget to update this in docs. `default` => `horizontal`
2. The penultimate grid can be filled in with default values, like 
| options: `horizontal` `vertical` | string | `horizontal` |  3.24.0   |

File: components/progress/demo/orientation.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+order: 0
+title:
+  zh-CN: 取向
+  en-US: Orientation
+---
+
+## zh-CN
+
+通过设置`orientaton =“ vertical”`，可以将方向从水平更改为垂直。
```
Comment (afc163) [inline components/progress/demo/orientation.md]:
```
通过设置 `orientaton="vertical"`，可以将方向从水平更改为垂直。
```

File: components/progress/demo/orientation.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+order: 0
+title:
+  zh-CN: 取向
```
Comment (afc163) [inline components/progress/demo/orientation.md]: 方向

File: components/progress/index.zh-CN.md (hunk 2 above)
Comment (afc163) [inline components/progress/index.zh-CN.md]: , => ，

File: components/progress/demo/orientation.md (hunk 3 above)
Comment (MrHeer) [reply components/progress/demo/orientation.md]:
```suggestion
通过设置 `orientation="vertical"`，可以将方向从水平更改为垂直。
```

File: components/progress/demo/orientation.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  en-US: Orientation
+---
+
+## zh-CN
+
+通过设置`orientaton =“ vertical”`，可以将方向从水平更改为垂直。
+
+## en-US
+
+By setting `orientaton="vertical"`, you can change the orientantion from horizontal to vertical.
```
Comment (MrHeer) [inline components/progress/demo/orientation.md]:
```suggestion
By setting `orientation="vertical"`, you can change the orientnation from horizontal to vertical.
```

File: components/progress/demo/orientation.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+---
+order: 0
```
Comment (MrHeer) [inline components/progress/demo/orientation.md]:
```suggestion
order: 11
```

**Review by MrHeer — CHANGES_REQUESTED** [review]

File: components/progress/index.zh-CN.md (hunk 2 above)
Comment (Larissagilliane) [reply components/progress/index.zh-CN.md]: Done

File: components/progress/demo/orientation.md (hunk 3 above)
Comment (Larissagilliane) [reply components/progress/demo/orientation.md]: Done

File: components/progress/demo/orientation.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 title:
-  zh-CN: 取向
+  zh-CN: 方向
   en-US: Orientation
 ---
 
 ## zh-CN
 
-通过设置`orientaton =“ vertical”`，可以将方向从水平更改为垂直。
+通过设置`orientaton=“vertical”`，可以将方向从水平更改为垂直。
```
Comment (orzyyyy) [inline components/progress/demo/orientation.md]:
```diff
- 通过设置`orientaton=“vertical”`，
+ 通过设置 `orientaton="vertical"`，
```

**Comment by VIRGO96** [comment]
has this been implemented in to progress bar yet ? anyone

**Comment by afc163** [comment]
ping~

**Comment by willwearing** [comment]
Hey team, 

Did this ever get implemented? Doesn't look like it from the documentation... 

Would be a really cool feature to have If not

**Comment by ErickLee-Spok** [comment]
^^

**Comment by balboFK** [comment]
@afc163 Hello again! 
Can we get a resoning behing why this was never merged?

File: components/progress/demo/orientation.md (hunk 7 above)
Comment (zaki1991) [reply components/progress/demo/orientation.md:10]: @Larissagilliane @afc163 Progress 没有找到 orientation 这个属性呢

**Comment by Novol1273** [comment]
So what was the final decision?
Will it ever be implemented?
Does it make sense to wait for this option to be added, or will I have to hardcode your components again?

**Comment by afc163** [comment]
1. This PR mixed unrelated commits that need to be clean.
2. We lost contact with PR author and can't get response.
3. PR is welcome to get it done.

@balboF @Novol1273

**Pull request closed without merging**